tracer/src/Datadog.Trace/RemoteConfigurationManagement/Protocol/GetRcmRequest.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="GetRcmRequest.cs" company="Datadog">
```

Comment:
Do you have a spec for this protocol that you could link somewhere?
Also, do you think it would clarify things if we were to divide the request side and response side in separate namespaces?